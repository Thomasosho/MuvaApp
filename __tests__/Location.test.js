jest.useFakeTimers("modern");
import React from "react";
import renderer from "react-test-renderer";
import LocationScreen from "../src/screens/location/Location";

it("renders correctly", () => {
  const tree = renderer.create(<LocationScreen />).toJSON();
  expect(tree).toMatchSnapshot();
});
