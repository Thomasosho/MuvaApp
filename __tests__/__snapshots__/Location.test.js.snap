// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Array [
      Object {
        "backgroundColor": "#fff",
        "flex": 1,
        "paddingHorizontal": 16,
        "paddingVertical": 16,
      },
      undefined,
    ]
  }
>
  <Text>
    LocationScreen
  </Text>
</View>
`;
